modules/math/src/main/java/com/opengamma/strata/math/impl/statistics/descriptive/ExponentiallyWeightedInterpolationQuantileMethod.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ * 
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.math.impl.statistics.descriptive;
+
+import com.opengamma.strata.collect.ArgChecker;
+import com.opengamma.strata.collect.DoubleArrayMath;
+import com.opengamma.strata.collect.array.DoubleArray;
+import com.opengamma.strata.math.impl.statistics.descriptive.QuantileCalculationMethod;
+
+/**
+ * Implementation of a quantile and expected shortfall estimator for series with exponentially weighted probabilities.
+ * <p> 
+ * Reference: "Value-at-risk", OpenGamma Documentation 31, Version 0.2, January 2016. Section A.4.
+ */
+public final class ExponentiallyWeightedInterpolationQuantileMethod
+    extends QuantileCalculationMethod {
+  
+  /** The exponential weight. */
+  private final double lambda;
+
+  /**
+   * Constructor. 
+   * @param lambda  the exponential weight
+   */
+  public ExponentiallyWeightedInterpolationQuantileMethod(double lambda) {
+    this.lambda = lambda;
```

Comment:
Any constraint on the range of lambda? The formula calculating weights breaks down in some cases.